src/TLuaInterpreter.cpp

```diff
@@ -3265,202 +3265,322 @@ int TLuaInterpreter::setBackgroundImage( lua_State *L )
 
 int TLuaInterpreter::setLabelClickCallback( lua_State *L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if(! pHost) {
+        lua_pushstring( L, tr( "setLabelClickCallback: NULL Host pointer - something is wrong!" )
+                        .toUtf8().constData() );
         lua_error( L );
         return 1;
     }
-    else
-    {
-        luaSendText = lua_tostring( L, 1 );
-    }
-    string luaName="";
-    if( ! lua_isstring( L, 2 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+
+    QString labelName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "setLabelClickCallback: bad argument #1 type (label name as string expected, got %1!)" )
```

Comment:
Well we do have some established "patterns" for the text before the `(`...`)` - and, even __I__ do not now anticipate translating the function names so from a "support" point of view some key word recognition (for "value" and "type" along side the argument number) will probably go a long way to determining the sort of issues a user is having.  Of course, the users who do work out from an error message (in their own language) what the problem is, they will most likely not be the ones seeking help from others!

With the "Show Lua errors in console" option these _will_ show up in the main screen will they not?

The Mudlet enthusiast who rolls their own Mudlet from source will have access to the `.ts` files as part of the source code bundle so it will be possible for those interested in getting into cross-language support to fire up Qt Linguist and cross-check what other language users will see for each error message - in the same manner as coders can dive into Qt Creator and poke around in the source code.

As for not understanding the language putting off those who want to help - will we demand all Forum posts to be in English from the people originally __seeking__ help.  It would be nice from a support point of view but I'd be surprised if it works out like that...!